src/core/DamageHorrorCounter.ttslua

```diff
@@ -97,6 +104,11 @@ end
 
 function updateLabel()
   local uiColor, uiVal = getUiColorAndVal()
+  if val > 9 then
+    self.UI.setAttribute("xmlTxt", "fontSize", "350")
+  else
+    self.UI.setAttribute("xmlTxt", "fontSize", "700")
+  end
```

Comment:
Imo this should be part of the function that gathers the data for the UI (`getUiColorAndVal()`). Would of course need a rename then.